metals/src/main/scala/scala/meta/internal/worksheets/MdocEnrichments.scala

```diff
@@ -49,4 +50,15 @@ object MdocEnrichments {
     }
   }
 
+  /**
+   * Determines whether or not an evaluated worksheet statement summary is
+   * complete or not. If it is complete, it just returns the summary, otherwise
+   * it will chop off the last 3 chars and replace them with ... to signify
+   * continuation
+   */
+  def truncatify(statement: EvaluatedWorksheetStatement): String = {
+    if (statement.isSummaryComplete()) statement.summary()
+    else statement.summary().dropRight(3) + "..."
```

Comment:
What about using the horizontal ellipsis unicode character? I think we can also skip `dropRight()` in that case so that the full summary is preferved from MUnit
```suggestion
    else statement.summary().dropRight(3) + "…"
```

Reply:
Ahh, that's a great idea.